addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/binlist/BinRecord.java

```diff
@@ -32,7 +32,7 @@ public final class BinRecord {
     private final String category;
     private final String issuer;
 
-    BinRecord(String bin, String brand, String category, String issuer) {
+    public BinRecord(String bin, String brand, String category, String issuer) {
```

Comment:
It was necessary to expose this for testing passive scan rules which leverage it because commoblib and the BinList singleton aren't available to the scan rules during unit testing.

Reply:
Which is probably also the case for the Alert documentation generation script?

Reply:
Technically this is not required, you should be able to get an existing `BinRecord` through the `BinList` but fine leaving as is though might worth adding JavaDoc that says it's preferable to use `BinList`.

Reply:
Will add some JavaDoc.